frontend/appflowy_flutter/lib/workspace/application/appearance.dart

```diff
@@ -181,7 +181,7 @@ class AppearanceSettingsState with _$AppearanceSettingsState {
     double menuOffset,
   ) {
     return AppearanceSettingsState(
-      appTheme: AppTheme.fromName(themeName),
+      appTheme: AppTheme.fallback,
```

Comment:
Is this correct? I would assume the `fromName`would use the fallback if not found.

Or do we not have the themeName in the initial state?

Reply:
Initializing the theme is now async because it has to read files from disk. The initial state for the `AppearanceSettingsBloc` shouldn't be async, though. That's why I used the fallback here.

```
  static Future<AppTheme> fromName(String themeName) async {
    for (final theme in await themes) {
      if (theme.themeName == themeName) {
        return theme;
      }
    }
    throw ArgumentError('The theme $themeName does not exist.');
```

The full function to get the theme `fromName` also throws an error if the theme name isn't found. The theme should always be resolved, if it isn't it should be considered unexpected behavior. The user should never be presented with an option to select a theme that doesn't exist.